src/tools/pub/cmd/pub-add.md

```diff
@@ -26,65 +26,151 @@ For example, if 0.13.3 is the latest stable version of the `http` package,
 then `dart pub add http` adds
 `http: ^0.13.3` under `dependencies` in the pubspec.
 
-To add a [dev dependency][], use the `--dev` option:
-
-[dev dependency]: /tools/pub/dependencies#dev-dependencies
+You can also specify a constraint or constraint range
 
 ```terminal
-$ dart pub add --dev test
+$ dart pub add foo:2.0.0
+$ dart pub add foo:'>=2.0.0'
```

Comment:
Maybe specify as `foo:'^2.0.0'`?

Reply:
Haha I didn't even think of that. Much better.